collection-scripts/gather_services_status

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
Minor nit

We are using #!/usr/bin/bash vs #!/bin/bash vs #!/usr/bin/env bash in different files, i think we keep this consistent everywhere unless we have a reason not to.